faqs/atm7_http.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to enable HTTP on ATM7
+search: atm7
+---
+
+**Singleplayer**
+1. Press the windows key and R at the same time
+2. Type in ``%appdata%\.minecraft``
+3. Go into ``saves`` then the name of your save/world
+4. Then go into serverconfig and search for ``computercraft-server.toml``
+5. Open the file in notepad or any other text editor and search for ``[http]``
+6. There you should find ``enabled``, that should be set to false
+7. Set the option to true and save the file
+8. You're done
+
+**Multiplayer**
+(If you're a player)
+1. Ask the server owner or someone who has access to the files to activate http
+2. Hope they do it and not deny it cause of "security"
+
+(If you're a server owner)
```

Comment:
This entire section is wrong. The config file does not magically become a `.cfg` file just because it's a server.

There is a reason the file is called computercraft-***server***.toml.

Reply:
i'll change it, mistakes have been made